packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -197,12 +197,13 @@ export default class DatabaseService {
       })
       .then(({ availability, capabilities, prices }) => {
         availability.forEach((plan) => {
+          const beta = plan.status === 'BETA' ? '-beta' : '';
```

Comment:
move to constants file

Reply:
Fixed: https://github.com/ovh/manager/pull/5586/commits/f4c7cf9487f1745c9bc94a262c785528601b64f5